atlasdb-config/build.gradle

```diff
@@ -1,32 +1,41 @@
 apply from: "../gradle/publish-jars.gradle"
 apply plugin: 'org.inferred.processors'
+apply plugin: 'com.github.johnrengelman.shadow'
 
 apply from: "../gradle/shared.gradle"
 
+configurations {
+    explicitShadow
+    compile.extendsFrom(explicitShadow)
+    shadow.extendsFrom(explicitShadow)
```

Comment:
I think the ALPN jar (not desired to be in compile) might have been an issue here, though. Now investigating